pkgs/development/interpreters/boron/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
```

Comment:
Why aren't you using a typical fetchurl? The project provides tarballs:

https://sourceforge.net/projects/urlan/files/Boron/

Reply:
Personally I find it easier if I need to build some other rev for whatever reason.
I don't really mind changing it to tarball tbh, whatever gets this merged! Let me know.

Reply:
I believe the tarball is better for the purposed of Nixpkgs.

Sometimes an "officially released" tarball is different from a raw Git-fetched code. Many projects run cleanup and preparation scripts as part of "tarballing" process.
Indeed it can be indirectly seen in many Nix packages that use `autoreconfHook` - they almost always come from raw `fetchFromGitHub`s that do not create `configure` scripts but keep the autoconf machinery.

On the other hand, if the tarball contents are byte-identical to the raw git, I believe the tarball is a better bet because it requires less dependencies.
All else being the same, the tarball can be fetched using curl (in the innards of `fetchurl`) and unpacked with the typical tar-zip tools already provided by `stdenv`, whereas `fetchgit` brings git.

Reply:
done in https://github.com/NixOS/nixpkgs/pull/180865/commits/a8b27f7fd36d24d79913bf8c08933829ee426e29